parboiled/src/main/scala/org/parboiled2/Base64Parsing.scala

```diff
@@ -60,6 +64,18 @@ trait Base64Parsing { this: Parser =>
       }
     }
   }
+
+  private def base64(alphabet: CharPredicate, fillChar: Char, decoder: Decoder): Rule1[Array[Byte]] = {
```

Comment:
This could be public to promote other custom alphabets.  I'm not sure what other variants exist.  I chose to be conservative on public API, but don't feel strongly.

Reply:
Yes, nowadays I agree with your conservative stance here.